projects/next/pages/_app.js

```diff
@@ -11,16 +15,39 @@ class MyApp extends App {
 
         return (
             <Container>
-                <ApolloProvider client={apolloClient}>
-                    <Head>
-                        <link
-                            href="/static/parrot.gif"
-                            rel="icon"
-                            type="image/gif"
+                <ThemeProvider>
+                    <ApolloProvider client={apolloClient}>
+                        <Head>
+                            <link
+                                href="/static/parrot.gif"
+                                rel="icon"
+                                type="image/gif"
+                            />
+                        </Head>
+                        <Global
+                            styles={(theme) =>
+                                css`
+                                    * {
```

Comment:
Is this the idiomatic place to add global styles in a Next.js app? If not, update.

Reply:
Works for now.